src/shared/Microsoft.AzureRepos/AzureReposHostProvider.cs

```diff
@@ -83,7 +90,7 @@ public async Task<ICredential> GetCredentialAsync(InputArguments input)
             return credential;
         }
 
-        public virtual Task StoreCredentialAsync(InputArguments input)
+        public Task StoreCredentialAsync(InputArguments input)
```

Comment:
Oh, completely forgot to submit this question as part of my review: Is this a "while at it" change, sneaked into this PR "just because"?

Reply:
Yes. I (my IDE) noticed this is a redundant modifier, as nothing is overriding this method 😁